web/src/message_edit.ts

```diff
@@ -592,6 +613,11 @@ function edit_message($row: JQuery, raw_content: string): void {
     currently_editing_messages.set(message.id, $message_edit_content);
     message_lists.current.show_edit_message($row, $form);
 
+    const previous_height = resized_edit_box_height.get(message.id);
+    if (previous_height) {
+        autosize.destroy($(the($message_edit_content))).height(previous_height + "px");
+    }
```

Comment:
I think this algorithm will have problems if the browser was resized in between rerenders.

Probably the `resized_edit_box_height` map should just get cleared in the main window resize handler.